source/src/processing/cs_PowerSampling.cpp

```diff
@@ -427,6 +444,7 @@ void PowerSampling::powerSampleAdcDone(adc_buffer_id_t bufIndex) {
 	//		State::getInstance().set(CS_TYPE::STATE_POWER_USAGE, &_avgPowerMilliWatt, sizeof(_avgPowerMilliWatt));
 	int32_t slowAvgPowerMilliWatt = _slowAvgPowerMilliWatt;
 	State::getInstance().set(CS_TYPE::STATE_POWER_USAGE, &slowAvgPowerMilliWatt, sizeof(slowAvgPowerMilliWatt));
+	storeEnergyUsed();
```

Comment:
Code around this edit is happy to be cleaned up a little bit :-)
E.g. `PS_TEST_PIN_TOGGLE` as random macro with not even a semi-colon and commented out code.
What guarantees this code can't be reached before init?

Reply:
Cleaning up the power sampling class is very out of scope :P